application/src/main/java/org/opentripplanner/netex/mapping/FlexStopsMapper.java

```diff
@@ -140,7 +140,7 @@ List<RegularStop> findStopsInFlexArea(
     List<RegularStop> stops = stopsSpatialIndex
       .query(geometry.getEnvelopeInternal())
       .stream()
-      .filter(stop -> flexibleStopTransitMode == stop.getGtfsVehicleType())
+      .filter(stop -> flexibleStopTransitMode == stop.getVehicleType())
```

Comment:
What is this used for? Vehicle type information is not required, I don't know how much we should rely on that information in general.

Reply:
I just noticed this is in the netex mapping, perhaps it's required there and this is fine.

Reply:
I have no clue, I just renamed the very confusing method name - as you noticed, it is used for both GTFS and NeTEx so having GTFS as part of the name is wrong.

Reply:
It's used to show a stop's mode in the API. However, since it's not required we also deduce it from the patterns that visit the stop: https://github.com/opentripplanner/OpenTripPlanner/blob/09e2de79e913a931c90958d24352bd40d6797827/application/src/main/java/org/opentripplanner/transit/service/DefaultTransitService.java#L730-L739